app/src/main/java/org/systers/mentorship/view/fragments/TasksFragment.kt

```diff
@@ -53,7 +54,7 @@ class TasksFragment(private var mentorshipRelation: Relationship) : BaseFragment
                     } else {
                         rvTasks.apply {
                             layoutManager = LinearLayoutManager(context)
-                            adapter = TasksAdapter(taskViewModel.tasksList, markTask)
+                            adapter = TasksAdapter(activity!!, taskViewModel.tasksList, ::markTask, ::deleteTask)
                         }
```

Comment:
Please remove **deleteTask** function from the TasksAdapter parameter. We don't need this function here in this task